Dockerfile

```diff
@@ -16,7 +16,8 @@ ENV \
 
 # Get needed build dependencies
 RUN \
-  apt-get -qq update && \ 
+  apt-get -qq update && \
+  apt-get -qq upgrade && \
```

Comment:
This is a part of the builder stage, and has no effect on the final image. This apt upgrade has to be done in `stage 1`, below this line: https://github.com/enonic/docker-xp7/blob/f73fb1d7be1a98bbe9115b5755c457284545cbdc/Dockerfile#L120